### DISCUSSION TITLE
Steve Manuel, Benjamin Eckel - dylibso

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#169] [Announcements]
## Background

- https://twitter.com/extism
- https://extism.org/
- https://twitter.com/dylibso
- https://dylibso.com/

### Steve

- https://twitter.com/nilslice

### Ben

- https://twitter.com/bhelx
- https://bhelx.simst.im/

## Intro

Andrew: Hello, welcome to the devtools.fm podcast. This is a podcast about developer tools and the people who make them. I'm Andrew and this is my co-host Justin.

Justin: Hey everyone! Our guests today are Steve Manuel and Benjamin Eckel of dylibso. I originally heard of y'all through your work on extism, a wasm extension framework. Before we get into that though, would y'all like to tell our audience a little bit more about yourselfs and about dylibso?

## Topics

- WASM
- dylibso
- extism

## Questions

- [x] Why do you think WASM is so exciting?

### Extism

- [x] What is dylibso? 
- [x] What types of plugins systems does this allow me to create? What types of apps?f
- [x] What features of WASM are extism built upon that make it a good solution?
- [x] The real innovation here seems to be the common interfact for loading and executing WASM in other languages. Is it ahrd calling WASM modules in langages other than JS?
- [x] Are there any programs that use extisim today?

### Future

- [x] What is Dylibso? 
- [x] What are some thing you have planned for dylibso?
- [x] What do you think the future of WASM is?

**Comment by nilslice** [comment]
This is great. Thank you! One edit, is that the company name is "Dylibso".

**Reply by just-be-dev** [reply]
I got it updated, thanks!

**Comment by bhelx** [comment]
My Tooltips:

https://github.com/bytecodealliance/wizer
https://www.lunarvim.org/

**Comment by nilslice** [comment]
My Tooltips: 
https://ziglang.org
https://helix-editor.com

**Comment by just-be-dev** [comment]
Tooltips:
- [Everything once sensor](https://shop.everythingsmart.io/products/everything-presence-one-kit) ([explainer video](https://www.youtube.com/watch?v=cH5sJFk_2wo))
- [Threadboards](https://www.kickstarter.com/projects/3dprinteracademy/thread-boards-3d-printable-peg-boards-with-a-twist?ref=nav_search&result=project&term=thread)

**Comment by hipstersmoothie** [comment]
Mentions:

https://github.com/MAIF/otoroshi

**Comment by hipstersmoothie** [comment]
Tooltips:

- https://www.youtube.com/watch?v=zMf_xeGPn6s&ab_channel=Remix
